tests/testthat/test-estimate_KM.R

```diff
@@ -22,6 +22,7 @@
 #' T3.3 When no strata are specified, an artificial strata is created 'Overall'
 #' T3.4 When only 1 stratum is specified, the stratum names are added to the `names` attribute'
 #' T3.5 When more than 1 strata is specified, the stratum names are available in the `names` attribute
+#' T3.6 The strata label is constructed correctly
```

Comment:
I'm not in favor of general descriptions. Can you explain what you mean with 'correctly'?
Perhaps: The strata labels are also available in a separate list element of the survfit object.

You also need to write a test to ensure:
- that this list element is added to the object.
- that the content is equal to the label of the corresponding variable in the data
- test if combination of labels is done as expected
- test if Overall is used if ~1 in formula

Reply:
All of these are tested already, although not each an explicit separate test. 
- that this list element is added to the object: already testing equality to a list object, there it must be a list.
- that the content is equal to the label of the corresponding variable in the data: this is the test that already appears
- that the content is equal to the names attribute of the strata element of the survfit object: does not apply, those are strata levels, not the strata labels
- test if combination of labels is done as expected: tested in the visr file, where the strata label is checked
- test if Overall is used if ~1 in formula: also already in the visr test file

Reply:
@SHAESEN2 to look into this again in final review.